src/StripeTests/Services/BankAccounts/BankAccountServiceTest.cs

```diff
@@ -26,19 +26,7 @@ public BankAccountServiceTest(MockHttpClientFixture mockHttpClientFixture)
 
             this.createOptions = new BankAccountCreateOptions
             {
-                SourceBankAccount = new SourceBankAccount
-                {
-                    AccountNumber = "000123456789",
-                    Country = "US",
-                    Currency = "usd",
-                    AccountHolderName = "John Doe",
-                    AccountHolderType = BankAccountHolderType.Company,
-                    RoutingNumber = "110000000",
-                    Metadata = new Dictionary<string, string>
-                    {
-                        { "key", "value" },
-                    },
-                }
+                SourceToken = "btok_123",
```

Comment:
Ah sorry, I didn't foresee it would fail current library tests. Yup, it is my work on typed params to make public spec for these params only taking String as an attempt to limit non-PCI compliant users from accidentally passing in raw PAN data. So as I understand, we want to make this enforcement on in the client lib too, so passing String here would be okay.

Just a side note: on live request OpenAPI validation, we skipped validation on these undocumented polymorphic param so we won't see false rate of validation failure. (Additionally, there's no sever side effect because its' OpenAPI)